deploy-queue/src/main.rs

```diff
@@ -92,24 +96,18 @@ async fn main() -> Result<()> {
         } => {
 
             // Insert deployment record into database
-            let deployment_id = match insert_deployment_record(
+            let deployment_id = insert_deployment_record(
                 &db_client,
-                Deployment {
-                   region,
-                   component,
+                &Deployment {
+                   region: region.clone(),
+                   component: component.clone(),
                    environment: environment.to_string(),
-                   version,
-                   url,
-                   note,
+                   version: version.clone(),
+                   url: url.clone(),
+                   note: note.clone(),
                    ..Default::default()
                 }
```

Comment:
I *think* this cloning isn't necessary, but I might be wrong. I'll take a look locally.